src/explore-education-statistics-admin/src/pages/release/data/components/ImporterStatus.tsx

```diff
@@ -22,11 +22,15 @@ export const getImportStatusLabel = (
       return 'Queued';
     case 'PROCESSING_ARCHIVE_FILE':
       return 'Processing archive file';
-    case 'RUNNING_PHASE_1':
+    case 'STAGE_1':
       return 'Validating';
-    case 'RUNNING_PHASE_2':
+    case 'STAGE_2':
       return 'Importing';
-    case 'RUNNING_PHASE_3':
+    case 'STAGE_3':
+      return 'Importing';
+    case 'STAGE_4':
+      return 'Importing';
+    case 'STAGE_5':
```

Comment:
Think we can remove STAGE_5 from the frontend now as well

Reply:
Oops, totally forgot that one, thanks.